vagrant/ansible/roles/tests/tasks/log.yml

```
@@ -0,0 +1,16 @@
+---
+- name: Log the line
+  delegate_to: localhost
+  become: true
+  block:
+    - name: Get current time
+      command: date "+%Y-%m-%d %H-%M-%S.%6N"
+      register: now
+
+    - name: Write log line
+      lineinfile:
+        dest: /var/log/site-tests.log
```

How about having separate log files for different tests? May be 
```suggestion
        dest: /var/log/log.{{ role }}
```

With #41 we could also think about having it under `{{ config.statedir }}`:
```suggestion
        dest: {{ config.statedir }}/log.{{ role }}
```

---

I'm ok with using `config.statedir`. However I don't see the benefit of having a different log file for each test. This log is really small, and used just to know when each test started and ended, and if it failed or not (and when). The idea is to be able to track times from global logs and map them to specific tests if necessary. Note that a "test" here represents a black box that can do anything, like sit-test-cases, which does a lot of smaller tests internally, but those are not seen nor logged by the sit-environment.

Each test will be able to generate its own logs if necessary, and I plan to incorporate monitoring of cpu, memory and other resources which will be independent per test.

---

Fine.